efs.tf

```diff
@@ -62,44 +68,42 @@ data "aws_iam_policy_document" "efs" {
   }
 }
 
+# EFS file system policy
+data "aws_iam_policy_document" "efs_file_system_policy" {
+  statement {
+    sid     = "EFS-CSI-Driver-Access"
+    effect  = "Allow"
+    actions = ["elasticfilesystem:ClientMount", "elasticfilesystem:ClientWrite", "elasticfilesystem:ClientRootAccess"]
+    principals {
+      type        = "AWS"
+      identifiers = [var.efs_node_iam_role_arn]
+    }
+    condition {
+      test     = "Bool"
+      values   = ["true"]
+      variable = "elasticfilesystem:AccessedViaMountTarget"
+    }
+  }
+}
+
 module "efs" {
-  source  = "terraform-aws-modules/efs/aws"
-  version = "1.6.3"
+  source  = "cloudposse/efs/aws"
+  version = "1.1.0"
+
+  region  = var.region
+  vpc_id  = var.vpc_id
+  subnets = var.private_subnets_id
 
-  name = "${var.cluster_name}-efs"
+  allowed_cidr_blocks   = var.private_subnets_cidrs
+  create_security_group = true
+  name                  = "${var.cluster_name}-efs"
 
-  mount_targets                      = local.mount_targets
   security_group_description         = "${var.cluster_name} EFS"
-  security_group_vpc_id              = var.vpc_id
-  attach_policy                      = true
   bypass_policy_lockout_safety_check = false
-  policy_statements = [
-    {
-      sid     = "EFS-CSI-Driver-Access"
-      actions = ["elasticfilesystem:ClientMount", "elasticfilesystem:ClientWrite", "elasticfilesystem:ClientRootAccess"]
-      principals = [
-        {
-          type        = "AWS"
-          identifiers = [var.efs_node_iam_role_arn]
-        }
-      ]
-      conditions = [{
-        test     = "Bool"
-        values   = ["true"]
-        variable = "elasticfilesystem:AccessedViaMountTarget"
-      }]
-    }
-  ]
-  throughput_mode      = var.throughput_mode
-  performance_mode     = var.performance_mode
-  enable_backup_policy = var.enable_backup_policy
-  security_group_rules = {
```

Comment:
These security group rules don't need to be specified in the other module?

Reply:
it is specified in line number 74